bin/run.js

```diff
@@ -805,7 +805,20 @@ const parseRcFile = path => {
   let contents
   try {
     contents = fs.readFileSync(path, 'utf8')
-  } catch (er) {
+  } catch {
+    try {
+      if (fs.accessSync(path + '.yaml') === undefined) {
```

Comment:
Why the access checks?  Just read the file, if it throws, it's not there or you can't read it.

Reply:
Okay. I have updated the PR accordingly.